cs464-group-project/src/components/Api/ApiRequest.js

```diff
@@ -41,3 +41,27 @@ export async function getAllTeamNames() {
     throw error;
   }
 }
+
+let allTeamLogos = null;
+
+export async function getTeamLogos() {
+  //if empty do the call
+  if (allTeamLogos === null) {
+    try {
+      const data = await getAllTeamNames();
+      allTeamLogos = data.teams.map((team) => ({
+        name: team.strTeam,
+        logo: team.strTeamLogo,
+      }));
```

Comment:
If just using logos and nothing else, this should work.  But I would include the idTeam prop as well.

Reply:
I've added idTeam in the allTeamLogos array.  I think when ever we might need to share api calls, we can just create new objects for it.  So we can just write our own stuff like the way we wrote our own API calls.